samcli/commands/build/build_context.py

```diff
@@ -508,3 +509,24 @@ def _is_layer_buildable(layer: LayerVersion):
             LOG.debug("Skip building pre-built layer: %s", layer.full_path)
             return False
         return True
+
+    _JAVA_BUILD_WARNING_MESSAGE = (
+        "Test the latest build changes for Java runtime 'SAM_CLI_BETA_MAVEN_SCOPE_AND_LAYER=1 sam build'. "
+        "These changes will replace the existing flow on 1st of April 2022. "
+        "Check __issue_line__ for more information."
+    )
+
+    def _check_java_warning(self):
```

Comment:
nit: I know we don't add type hints and docstrings to older files like this, but should we start for consistency going forward?

Reply:
Good point, I will update the PR